### PR TITLE
Adds Cargo Departmental Console

### PR DESCRIPTION
**Pull request by Penwin0**
## About The Pull Request
Adds the cargo departmental console wich allows the QM to demote his underlings or add access to cargo

## Why It's Good For The Game
The QM is a head in every other aspect it only makes sense for him to be able to demote, also helps in rounds with no Cap/HoP

## Changelog
:cl:
add: Added cargo departmental console
:/cl:

### CODE REVIEW

File: code/game/machinery/computer/card.dm [hunk 1]
```diff
@@ -391,6 +391,9 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 						if((ACCESS_CE in scan.access) && ((target_dept==5) || !target_dept))
 							region_access |= 5
 							get_subordinates("Chief Engineer")
+						if((ACCESS_QM in scan.access) && ((target_dept==6) || !target_dept))
```
Comment (MACIEKBAKI) [inline code/game/machinery/computer/card.dm:395]: Mark this as a toolbeltstation change

Comment (MACIEKBAKI) [reply code/game/machinery/computer/card.dm:395]: The two lines under it, too

File: code/game/machinery/computer/card.dm [hunk 2]
```diff
@@ -610,7 +613,7 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 		typed_circuit.target_dept = target_dept
 	else
 		target_dept = typed_circuit.target_dept
-	var/list/dept_list = list("general","security","medical","science","engineering")
+	var/list/dept_list = list("general","security","medical","science","engineering","cargo")
```
Comment (MACIEKBAKI) [inline code/game/machinery/computer/card.dm:617]: maybe mark as a change?

File: code/game/machinery/computer/card.dm [hunk 3]
```diff
@@ -634,3 +637,9 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 	icon_screen = "idce"
 
 	light_color = LIGHT_COLOR_YELLOW
+
+/obj/machinery/computer/card/minor/qm
+	target_dept = 6
+	icon_screen = "idce"
+
+	light_color = LIGHT_COLOR_YELLOW
```
Comment (MACIEKBAKI) [inline code/game/machinery/computer/card.dm:646]: I'd much rather this be in a seperate file under toolbeltstation/code/game/machinery/computer/card.dm, but if you place it here atleast mark this as a toolbeltstation addition

File: code/game/machinery/computer/card.dm (hunk 1 above)
Comment (Penwin0) [reply code/game/machinery/computer/card.dm:395]: Alright will do, will probably move it to toolbelt folder in another PR down the line, will just comment it for now

File: code/game/machinery/computer/card.dm (hunk 3 above)
Comment (Penwin0) [reply code/game/machinery/computer/card.dm:646]: Read other comment

File: code/game/machinery/computer/card.dm (hunk 2 above)
Comment (Penwin0) [reply code/game/machinery/computer/card.dm:617]: Read other comment

**Comment by MACIEKBAKI** [comment]
Why do you add a department console on a branch called fix_autolathe

**Comment by Penwin0** [comment]
Cause originally i was going to fix the autolathe thingy, but i dont know how to fix it so i just made this instead

**Comment by Penwin0** [comment]
ok is fix @MACIEKBAKI

**Pull request merged**